acl/aclauthz/acl_cache.go

```diff
@@ -264,3 +266,54 @@ func (c *ACLCaches) Purge() {
 		}
 	}
 }
+
+// TODO: unexport once this is moved into the same package as ACLResolver
+func NewPolicyAuthorizerWithCache(policies structs.ACLPolicies, cache *ACLCaches, entConf *acl.Config) (acl.Authorizer, error) {
```

Comment:
This was `ACLPolicies.Compile` , the only change here is to the function signature and the call to `resolveWithCache` since that function signature also changed.